frontend/src/scss/PrimeStyles.scss

```diff
@@ -789,3 +805,47 @@ $results-dropdown-spacing: #{units(4)} - #{units(2)} - 22px - #{units(4)}; // he
   padding: units(2.5) units(3);
   position: relative;
 }
+
+.modal__footer--sticky {
+  @include at-media(mobile) {
+    z-index: 1;
+    position: absolute;
+    width: 100%;
+    right: 0;
+    bottom: 0;
+    padding: units(2.5);
+    box-shadow: 0 units(-0.5) units(1) 0 rgba(0, 0, 0, 0.1);
+    background: color("white");
+  }
+
+  @include at-media(desktop) {
+    position: fixed;
+    width: unset;
+    right: unset;
+    padding: 0;
+    bottom: units(7);
+  }
+}
+
+.modal__submit--container {
+  @include at-media(mobile) {
+    z-index: 1;
+    position: absolute;
+    width: 100%;
+    right: 0;
+    bottom: 0;
+    padding: units(2.5);
+    background: color("white");
+    margin-left: 0 !important;
+    margin-right: 0 !important;
+  }
+
+  @include at-media(desktop) {
+    box-shadow: unset;
+    margin: unset;
+  }
+}
+
+.section__qr--container {
```

Comment:
`section` is not a descriptive name, since it's specific to the AOE modal, `aoe-modal__qr` should work

Reply:
Thanks for the review! will fix it on my morning

Reply:
> this doesn't seem right:
> <img alt="Screen Shot 2021-05-18 at 11 40 43 AM" width="813" src="https://user-images.githubusercontent.com/5249443/118706218-f4e8ac80-b7cd-11eb-9605-4a3a7290f882.png">

Can you please describe what the actual size it is? Can't reproduce it on my side